heron/api/src/java/BUILD

```diff
@@ -14,6 +23,7 @@ api_deps_files =  \
 java_library(
     name = "api-java",
     srcs = glob(["com/twitter/heron/api/**/*.java"]),
+    javacopts = SOURCE_8_TARGET_8 + DOCLINT_HTML_AND_SYNTAX,
```

Comment:
we should apply javac settings consistently for all targets. Can we do this for all?